create_split.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import fire
+from typing import TypedDict, Set, List, Tuple
+from pathlib import Path
+import random
+
+random.seed(42_2137)
+
+
+class DatasetSplit(TypedDict):
+    train: Set[str]
+    test: Set[str]
+    eval: Set[str]
+
+
+DISEASES_TRAIN: Set[str] = set(["Diseases caused by viruses"])
```

Comment:
If we will not use hardcoded train, test and eval sets, we might be able to make this code generic to split any future dataset.

Reply:
I'll make an issue about refactoring dataset related stuff, for now lets leave it like that